driver-mct/shr/seq_flds_mod.F90

```diff
@@ -1387,6 +1387,15 @@ subroutine seq_flds_set(nmlfile, ID, infodata)
     attname  = 'Faxx_evap'
     call metadata_set(attname, longname, stdname, units)
 
+    if (trim(cime_model) == 'e3sm') then
+       call seq_flds_add(l2x_states,"Flrl_wslake")
+       longname = 'Lake water storage flux'
+       stdname  = 'lake_water_storage_flux'
+       units    = 'kg m-2 s-1'
+       attname  = 'Flrl_wslake'
+       call metadata_set(attname, longname, stdname, units)
+    end if
+
     ! Dust flux (particle bin number 1)
```

Comment:
@mabrunke - we no longer share this code with cesm so the check for cime_model is unnecessary and probably should be removed